python/Cargo.toml

```diff
@@ -28,11 +28,11 @@ geoparquet = { path = "../rust/geoparquet" }
 geozero = "0.14"
 http-range-client = { version = "0.9.0", default-features = false }
 indexmap = "2.5.0"
-numpy = "0.26"
+numpy = "0.27"
 object_store = "0.12"
-parquet = "56"
-pyo3 = { version = "0.26", features = ["hashbrown", "serde", "anyhow"] }
-pyo3-arrow = "0.12"
+parquet = "57"
+pyo3 = { version = "0.27", features = ["hashbrown", "serde", "anyhow"] }
```

Comment:
Can we split the pyo3 bump and the arrow bump into two separate PRs?

Reply:
According to the [version compatibility matrix](https://github.com/kylebarron/arro3/tree/main/pyo3-arrow#version-compatibility), arrow bump seems to require pyo3 and pyo3-arrow bump simultaneously. I tried bump arrow and pyo3 separately, but both resulted in dependency conflicts causing compilation errors. Please correct me if I'm wrong.

Reply:
> According to the [version compatibility matrix](https://github.com/kylebarron/arro3/tree/main/pyo3-arrow#version-compatibility)

Use v0.14 which uses latest arrow-rs but pyo3 0.26

Reply:
Thanks. I rolled back the pyo3 bump, retaining only the arrow bump.